draft-ietf-tls-tlsflags.xml

```diff
@@ -64,10 +64,10 @@
             <t> This document proposes a single extension called tls_flags that can enumerate such flag
                 extensions and allowing both client and server to indicate support for optional features in a
                 concise way.</t>
-            <t> None of the current proposed extensions are such that the server indicates support without the
-                client first indicating support. This specification enforces this restriction by specifying 
-                in <xref target="rules"/> that server bits may only reflect flags for which the client 
-                extension has already indicated support.</t> 
+            <t> None of the current proposed extensions are such that the extension receiver indicates support 
+                without the extension sender first indicating support. This specification enforces this restriction 
```

Comment:
I understand where you're going with this. You want to cover cases other than "client indicates support in ClientHello, and server indicates in ServerHello or EE".   I'm not sure that the text makes sense as stated. 

I couldn't find "extension sender" or "extension receiver" in RFC 8446. I have found "sender" and "receiver" of specific messages, but even if we extend the definition to cover extensions, support is only ever indicated by the sender. If the receiver indicates support it's in the context of another message, or rather, another flags extension.

Perhaps a better formulation would be : 
```
   None of the current proposed extensions allow for indication of 
   support in ServerHello (SH), EncryptedExtensions (EE), Certificate 
   (CT), or HelloRetryRequest (HRR) without first being indicated in 
   ClientHello (CH), and none of the current proposed extensions 
   allow for indication of support in the client-side Certificate (CT) 
   message without first being indicated in the server's 
   CertificateRequest (CR) message. This restriction is enforced by
   the rules in <xref target="rules"/>
```

Reply:
That works for me 👍 Want to make that change explicitly and submit?